packages/form/addon/components/cf-field/input/file.hbs

```diff
@@ -1,16 +1,29 @@
 {{#unless @disabled}}
-  <div class="uk-width-1-1" uk-form-custom="target: true">
+  <div uk-grid>
+    <div class="uk-width-expand" uk-form-custom="target: true">
 
-    <input
-      type="file"
-      name={{@field.pk}}
-      id={{@field.pk}}
-      {{on "change" this.save}}
-    />
+      <input
+        type="file"
+        name={{@field.pk}}
+        id={{@field.pk}}
+        {{on "change" this.save}}
+      />
 
-    {{! label is in another component }}
-    {{! template-lint-disable require-input-label }}
-    <input class="uk-input" type="text" placeholder={{@placeholder}} readonly />
+      {{! label is in another component }}
+      {{! template-lint-disable require-input-label }}
+      <input
+        class="uk-input"
+        type="text"
+        placeholder={{this.placeholder}}
```

Comment:
This means that the placeholder needs to be entered in the form builder, right? Would it make sense to add a generic fallback text (e.g. "Click to upload a file")? I think this would be fine for most use-cases.

Reply:
Misunderstanding, that's exactly what we're doing.